components/telemetry-operator/README.md

```diff
@@ -0,0 +1,65 @@
+# Telemetry Operator
+
+## Overview
+
+The telemetry operator contains a logging controller that generates a Fluent Bit configuration from one or more `LoggingConfiguration` custom resources. The controller ensures that all Fluent Bit pods run the current configuration by deleting pods after the configuration has changed. Find all CRD attributes [here](api/v1alpha1/loggingconfiguration_types.go) and an example [here](config/samples/telemetry_v1alpha1_loggingconfiguration.yaml).
+
+Creating Fluent Bit pods is for now out of scope of the operator and an existing Fluent Bit DaemonSet is expected.
```

Comment:
```suggestion
For now, creating Fluent Bit pods is out of scope of the operator. An existing Fluent Bit DaemonSet is expected.
```

What do you mean by "an existing ... is expected" - does it already exist or do we expect it to exist in the future?